app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown_menu.erb

```diff
@@ -50,7 +64,7 @@
 
         <li class="dropdown-menu-account-item">
           <%= link_to log_out_url, method: :delete, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-no_margin', title: 'Log Out of Forever' do %>
```

Comment:
Should we look at possibly disabling this link or change the path? When I am impersonating, I would expect that it would sign out of the person I am impersonating, not myself. And then if I log in myself again, i am still impersonating. 
That leads me to wonder also... there is no (correct) way to log out when you are impersonating.